src/Examples/UnitModels/Operations/skeleton_unit_testing.ipynb

```diff
@@ -0,0 +1,420 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# IDAES Skeleton Unit Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook demonstrates usage of the IDAES Skeleton Unit Model, which provides a generic \"bares bones\" unit for user-defined models and custom variable and constraint sets. To allow maximum versatility, this unit may be defined as a surrogate model or a custom equation-oriented model. Users must add ports and variables that match connected models, and this is facilitated through a provided method to add port-variable sets.\n",
+    "\n",
+    "For users who wish to train surrogates with IDAES tools and insert obtained models into a flowsheet, see the more versatile `SurrogateBlock()` for streamlined integration."
```

Comment:
This should be reworded. Just direct users to the general surrogate modeling docs rather than `SurrogateBlock`. For one, `SurrogateBlocks` do not handle the training of the surrogate, so the above statement is misleading at face value.